extensions/WSLExtension/Services/WslServicesMediator.cs

```diff
@@ -19,6 +20,8 @@ namespace WSLExtension.Services;
 /// </summary>
 public class WslServicesMediator : IWslServicesMediator
 {
+    private readonly ILogger _log = Log.ForContext("SourceContext", nameof(WslServicesMediator));

```

Comment:
Are you using the logger in here anymore?

Reply:
I don't think so, I removed it. Thanks